flow-model-generator/modules/flow-model-generator-core/src/main/java/io/ballerina/flowmodelgenerator/core/search/VectorStoreSearchCommand.java

```diff
@@ -0,0 +1,64 @@
+package io.ballerina.flowmodelgenerator.core.search;
+
+import io.ballerina.flowmodelgenerator.core.LocalIndexCentral;
+import io.ballerina.flowmodelgenerator.core.model.AvailableNode;
+import io.ballerina.flowmodelgenerator.core.model.Category;
+import io.ballerina.flowmodelgenerator.core.model.Item;
+import io.ballerina.modelgenerator.commons.SearchResult;
+import io.ballerina.projects.Project;
+import io.ballerina.tools.text.LineRange;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Handles the search command for vector stores.
+ *
+ * @since 1.0.0
```

Comment:
Check other places too
```suggestion
 * @since 1.1.0
```

Reply:
Fixed with [3f9272e](https://github.com/ballerina-platform/ballerina-language-server/pull/128/commits/3f9272ed5af482ef82b46b10ae0a087c58afa01f)